packages/react-composites/src/composites/CallComposite/styles/CallConfiguration.styles.ts

```diff
@@ -46,7 +52,7 @@ export const configurationContainerStyle = (desktop: boolean, backgroundImageUrl
   root: {
     height: '100%',
     width: '100%',
-    padding: '2rem 1rem 2rem 1rem',
+    padding: desktop ? '2rem 1rem 2rem 1rem' : '1rem 1rem 2rem 1rem',
```

Comment:
I think the padding on the top for mobile should be 1.5rem see image ->
![image](https://github.com/Azure/communication-ui-library/assets/94866715/48d5e18b-e496-49e5-9d28-a4141b0efd9b)

Reply:
This is also being affected 
![image](https://github.com/Azure/communication-ui-library/assets/94866715/90c06eae-8e70-4edd-8404-751716f47b1f)

Reply:
Thanks for checking these! Yep for mobile the reduced top padding is intentional to tighten up the spacing (particularly on limited landscape mobile), and desktop yep the increased distanced between title text and device section is intentional to match the 32px on figma
![image](https://github.com/Azure/communication-ui-library/assets/2684369/0d12d536-3b21-44ed-ba27-9e70574c4044)